### DISCUSSION TITLE
Apply style based on parent class?

### DISCUSSION THREAD

**Discussion by iannngit** [tailwindlabs/tailwindcss#9159] [Help] [answered]
Heylo y'all,

I'll use JS to toggle .scrolled on <body> and wonder if I can achieve something like this please? 
```
if <body> has .scrolled
then apply .bg-white to <header>
  and apply .text-black to text elements in <header>
```

**Comment by wongjn** [comment, marked as answer]
You can use [arbitrary variant](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants) for this:

```html
<header class="[body.scrolled_&]:bg-white [body.scrolled_&]:text-black">
  <!-- … -->
</header>

**Reply by iannngit** [reply]
I think the _ represents a space? Thank you JW!

**Reply by wongjn** [reply]
Yes, [as per the documentation](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants):

> If you need spaces in your selector, you can use an underscore.
